pkg/operator/ceph/csi/template/cephfs/csi-cephfsplugin.yaml

```diff
@@ -115,6 +120,10 @@ spec:
               mountPath: /tmp/csi/keys
             - name: host-run-mount
               mountPath: /run/mount
+            {{ if .CSILogRotation }}
+            - mountPath: {{ .CsiLogDirPath }}/cephcsi/log/cephfs-plugin
+              name: csi-log
+            {{ end }}
```

Comment:
I didn't see csi-addons sidecar with cephfs plugin daemon, is it intentional @Rakshith-R ?

Reply:
yes, there's no csiaddons server in cephcsi cephfs nodeplugin currently.